updateSL.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
```

Comment:
copy_from_graal.py
Also please add small comment in the header what this is good for.